students/nick_miller/lesson06/test_mailroom-pt4.py

```diff
@@ -52,11 +88,24 @@ def test_quit_prog():
 
 print("Function check:")
 if test_letter_prep() is None:
-    print("letter_prep() is good")
+    print("1. letter_prep() is good")
 if test_letter_format() is None:
-    print("letter_format() is good")
+    print("2. letter_format() is good")
+if test_input_prep() is None:
+    print("3. input_prep() is good")
+if test_thanks_all() is None:
+    print("4. thanks_all() is good")
+if test_list_check() is None:
+    print("5. list_check() is good")
+if test_input_check() is None:
+    print("6. input_check() is good")
+if test_db_update() is None:
+    print("7. db_update() is good")
+if test_save_file() is None:
+    print("8. save_file() is good")
 if test_report_sort_key() is None:
-    print("report_sort_key() is good")
+    print("9. report_sort_key() is good")
 if test_quit_prog() is None:
-    print("quit_prog() is good")
-
+    print("10. quit_prog() is good")
+print()
+print("10 tests run")
```

Comment:
you should not be doing this manually - you should be using `pytest` command to run all of your tests.

Reply:
were you able to run your tests via pytest?